### ISSUE TITLE
Ruby and Rails upgrades

### ISSUE THREAD

**Issue by mattr-** [gabriel-dehan/dyson-sphere-blueprints#62]
Hi! :wave:

I was looking at the source code for the site and noticed that both Ruby and Rails are a bit out of date. Would you be interested in having PRs to upgrade those? 

Those upgrades can sometimes be a lot of work, so I'm asking first since I wouldn't actually be on the hook for making sure the upgrades work as part of deploying a new version of the site. 😅

**Comment by gabriel-dehan** [comment]
Yeah the site is starting to get a lil' "old". I am all for an upgrade, but I'll probably have to do some extensive testing before the merge and deploy so I can't guarantee it'll be merged soon :p

**Comment by gabriel-dehan** [comment]
Forgot you had opened this issue 2 years ago, and I opened this one last week: #92 , oopsy :D
